client/src/pages/ForumPage/ForumPage.js

```diff
@@ -19,8 +22,32 @@ function pillClick(val) {
 const handleClick = e => {
   console.log("Clicked");
 };
+class ForumPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      topics: [],
+      hashtags: [],
+    }
+  }
+
+componentDidMount() {
+  axios.get(`${BASE_URL}/hashtags`)
+    .then(response => {
+      this.setState({ 
+        hashtags: response.data
+      })
+      console.log(response.data);
+    })
+    .catch(error => {
+      console.log('Error', error);
+    })
+}
```

Comment:
All API calls should take place in side the appropriate service in the `services/` directory. Can you move this there?

Reply:
Added Forum.js page in services/ directory and created import/export of axios fetch for hastags on ForumPage.js